src/components/comment-form/comment-form.twig

```diff
@@ -0,0 +1,29 @@
+<form
+  class="c-comment-form{% if class %} {{class}}{% endif %}"
+  {%- if action %}action="{{ action }}"{% endif -%}
+  {%- if method %}method="{{ method }}"{% endif -%}
+>
+  <h2>Leave a Comment</h2>
+  <p>Please be kind, courteous and constructive. You may use simple HTML or <a href="https://en.support.wordpress.com/markdown-quick-reference">Markdown</a> in your comments. All fields are required.</p>
+  <label>
+    <div class="c-comment-form__label">Message</div>
+    {% include '@cloudfour/components/input/input.twig' with { type: 'textarea', name: 'comment', required: true } only %}
+  </label>
+  {% if loggedInUser and logOutUrl %}
+    <p>Logged in as <a href="{{ loggedInUser.link() }}">{{ loggedInUser.name() }}</a>. <a href="{{ logOutUrl }}">Log out?</a></p>
+  {% else %}
+    <label class="c-comment-form__name">
+      <div class="c-comment-form__label">Name</div>
+      {% include '@cloudfour/components/input/input.twig' with { name: 'author', autocomplete: 'name', required: true } %}
+    </label>
```

Comment:
This may be overkill, but is there a low-level "label + input" layout object we might want to create for this recurring combination?

Reply:
Yeah, I was thinking about that while implementing this. Do you think it would make sense to modify the input component to have an optional (or required?) `label` prop that wraps the input in a label and stylizes the label?

Reply:
I guess it depends on whether or not it's simpler to create a separate layout object for that or to extend the input component template. I would traditionally opt for the former, but I'm more open than usual to the latter after [some recent experiences](https://cloudfour.com/thinks/mighty-morphin-web-components/).